packages/plugin-table/src/TableDomParser.ts

```diff
@@ -1,20 +1,129 @@
 import { AbstractParser } from '../../core/src/AbstractParser';
 import { DomParsingEngine } from '../../plugin-dom/src/DomParsingEngine';
 import { TableNode } from './TableNode';
+import { TableRowNode } from './TableRowNode';
+import { TableCellNode } from './TableCellNode';
 
 export class TableDomParser extends AbstractParser<Node> {
     static id = 'dom';
     engine: DomParsingEngine;
 
-    predicate = (item: Node): boolean => {
-        return item instanceof Element && item.tagName === 'TABLE';
+    predicate = (item: Node): item is HTMLTableElement => {
+        return item.nodeName === 'TABLE';
```

Comment:
Why did you change this line ? The only problem it had was that `Element` should have been changed to `HTMLTableElement`.